client/src/components/calender.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import BigCalendar from 'react-big-calendar';
+import 'react-big-calendar/lib/css/react-big-calendar.css'
+import EventForm from './eventForm.jsx'
+var moment = require('moment');
+
+BigCalendar.momentLocalizer(moment);
+moment().format();
+
+moment("2010-10-20 4:30", "YYYY-MM-DD HH:mm"); 
+
+class Calendar extends React.Component {
+  constructor(props) {
+  	super(props);
+  	this.state = {
+  	  formStatus: false
+  	}
+  	this.handleOpen = this.handleOpen.bind(this);
+  }
+
+  handleOpen () {
+  	console.log('hello');
+    this.setState({formStatus: true});
+    console.log(this.state.formStatus)
+  }
+
+  render(){
+    return (
+      <div {...this.props}>
+        <h3 className="callout">
+          Click an event to see more info, or
+          drag the mouse over the calendar to select a date/time range.
+        </h3>
+        <EventForm formStatus={this.state.formStatus}/>
+        <BigCalendar
+          selectable
+          events= {events}
+          defaultView='month'
+          scrollToTime={new Date(1970, 1, 1, 6)}
+          defaultDate={new Date(2017, 8, 26)}
+          //onSelectEvent={}// pull up info for event, sign up button included
```

Comment:
This is an event handler I want to use later, but the component will not render if it is empty.